tests/btb/test_hyperparameter_old.py

```diff
@@ -1,15 +1,15 @@
 import copy
```

Comment:
@csala I see that you renamed test_hyperparameter.py to test_hyperparameter_old.py. Is there a new set of unit tests for the HyperParameter classses? If so I don't see them in the PR.

Reply:
No! I actually renamed it to `_old` before creating the new test files but at the end decided to not rewrite the ones for this module and forget about it. I'll rename it.